results-processor/test_server.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 The WPT Dashboard Project. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import logging
+import sys
+
+import flask
+
+logging.basicConfig(level=logging.ERROR, stream=sys.stdout)
+app = flask.Flask(__name__)
+
+
+@app.route('/api/screenshots/upload', methods=['POST'])
```

Comment:
I'm somewhat surprised to see a push model; are we going to be underneath the appengine restrictions on upload file sizes?

Reply:
If by push you mean the processor uploading screenshots to `/api/screenshots/upload` (which is in the webapp), then yes it's push. The `MAXIMUM_BATCH_SIZE` is small enough to make sure we are always far away from the request size limit (100 * 100KB = 10MB < 32MB, and rarely we see screenshots close to 100KB). I could implement a higher but flexible (based on sizes) batch size, but that'd be an overkill given batch size of 100 already gives us superb performance.